services/web-app/pages/about-carbon/how-carbon-works.mdx

```diff
@@ -67,25 +67,26 @@ as color tokens, type tokens, spacing tokens, the grid, and more.
 **Guidelines** provide direction on critical topics such as accessibility and content and ensure
 consistent user experiences that meet IBM's standards.
 
-**Assets** include re–usable resources such as components, patterns, functions, and templates. Each
-asset has code, designs, and usage guidance.
+**Assets** are re–usable resources such as components, patterns, functions, and templates. Assets
+often have usage and accessibililty guidance, as well as styling specifications and code.
 
-**Design kits** are libraries from Sketch and Figma—and other tools such as Adobe XD and Invision.
-They contain components and elements as well as wireframes and guidance to aid designers.
+**Design kits** contain elements, guidelines, user interface components and patterns, and wireframes
+for various design tools as the design counterpart to coded assets.
 
-**Libraries** are 1:1 with code packages. All coded assets belong to a library and have a
-maintainer. Design kits with compatible code also live in libraries.
+**Libraries** are the means to contribute, install, and use one or many assets. All coded assets
+belong to a library. Most all libraries are published as code packages. Libraries have compatible
+design kits.
 
-## Carbon maintainers
+## Maintainers
 
-Carbon core and local systems have `maintainers` that administer governance over content and
-resource visibility. Each resource cites a maintaining team if an IBM team primarily maintains the
-resource. If an individual or informal group is responsible for a resource, it will be listed as
-community maintained.
+Carbon systems have maintainers that administer governance over content and resource visibility.
+Design kits and libraries cite a maintaining team if an IBM team primarily maintains the resource.
+If an individual or informal group is responsible for a resource, it will be listed as community
+maintained.
 
-Maintainers can control content and resource visibility through a metadata file that follows a clear
-`schema`. The [schema](/contributing/schema) helps us document each resource in a standardized way
-and enables everything to be indexed by Carbon’s new underlying platform.
+Maintainers can control content and resource visibility through a metadata file that follows a
+[schema](/contributing/schema). The schema helps us document each resource in a standardized way and
+enables everything to be indexed by Carbon’s underlying platform.
 
-The Carbon platform and website is maintained by the Carbon team. Progress on the new platform can
-be found in our [roadmap](/about-carbon/platform-roadmap).
+The Carbon platform and website is maintained by the Carbon Platform Squad. Progress on the new
```

Comment:
More of a broader comment, but we mention our team squads a lot on Platform. I don't think a lot of people will understand who and what our squads are. I think we should avoid squad naming and just say "the carbon core team" or whatever our larger team name is.

It feels a little bit like showing how the sausage is made.

Reply:
Can you open a separate issue if you want to explore that change? Currently, we have the two Carbon core squads defined, where each has its own icon and pictogram. That information is shown in the catalog cards and library and asset detail dashboards.

<img width="306" alt="image" src="https://user-images.githubusercontent.com/1691245/207387244-ff288998-c252-4801-b5c2-08fd7a5fbc5e.png">

The solution to bring utmost clarity is this issue, where we can put individual maintainer names on a per-resource basis.

https://github.com/carbon-design-system/carbon-platform/issues/295

Reply:
I think we should leave it at Carbon team as we went into the content of this page with the intention of not focussing on our team but explaining the broader ecosystem. 'Squad' feels like business jargon to the average reader.

Reply:
I opened a stub issue so we don't forget about this.
https://github.com/carbon-design-system/carbon-platform/issues/1547

Reply:
@aubrey-oneal to not add greater emphasis of squads through this PR, I will back out this change so it's just the Carbon team that maintains the platform.

Since we're currently specifying `Carbon System Squad` and `Carbon Dotcom Squad` at the library and asset level... I figured it'd be the most consistent and transparent to stick to that naming pattern.